### DISCUSSION TITLE
Outer deadzones

### DISCUSSION THREAD

**Discussion by Sumpftier** [cathery/sys-con#148] [Feature requests]
**Is your feature request related to a problem? Please describe.**
my xbone elite controller has an outer deadzone problem because of a mod

**Describe the solution you'd like**
add an outer deadzone feature to the config

**Comment by cathery** [comment]
You can configure the deadzone of Xbox One controllers in `sdmc:/config/sys-con/config_xboxone.ini`. Unless you mean something else by "outer deadzone"

**Comment by Sumpftier** [comment]
outer deadzones are basically inverted deadzones.
They let you reach the stick borders without actually reaching them if you set the deadzone high enough
https://gamefaqs.gamespot.com/boards/895574-gears-of-war-4/74407032

**Comment by Sumpftier** [comment]
btw I bought my elite controller just because of this amazing mod!
keep up the great work! :)

**Comment by cathery** [comment]
I'll see what i can do about the outer deadzones

**Comment by Sumpftier** [comment]
btw I bought my elite controller just because of this amazing mod!
keep up the great work! :)
